content/departments/product-engineering/process/sdlc.md

```diff
@@ -0,0 +1,43 @@
+# Software Development Life Cycle
+
+Sourcegraph uses two types of approaches to drive changes:
```

Comment:
The goal of the doc is to be high level, so I initially left them out. We can mention ticketing systems as an alternative to tracking work items.

Reply:
I think it's worth calling out still - after reflecting on this a little, a lot of major initiatives still start as GitHub issues or even just a straight-up pull request. For example, reworking database testing across the entire codebase (a high-level, far-reaching change) started as a lone pull request: https://github.com/sourcegraph/sourcegraph/pull/26129

Same with the recent effort to overhaul migrations development, also a pretty high-level and far-reaching change: https://github.com/sourcegraph/sourcegraph/pull/30276 and https://github.com/sourcegraph/sourcegraph/pull/29831

Significant portions of RFCs that I think are fairly high-level are also developed and discussed in issues: https://github.com/sourcegraph/sourcegraph/issues/12010 (maybe not the best example, but I remember writing this and I'm sure there are more)

It might even be worth calling out discussions, e.g. this ~60-comment discussion designing the Code Insights API: https://github.com/sourcegraph/sourcegraph/discussions/25201